rook/initialise.go

```diff
@@ -465,3 +459,23 @@ func doTemplate(input string, answers Answers) (output string, err error) {
 	output = out.String()
 	return
 }
+
+func generateEntryCode(filename string, entryType string) (err error) {
+	buf := bytes.Buffer{}
+
+	buf.WriteString(`// generated by "sampctl init"`)
```

Comment:
this could actually just be a single WriteString call, this was my fault for writing it in such a weird way at the start.

\` string literals in Go can span multiple lines:

```
buf.WriteString(`// generated by "sampctl init"

#include <a_samp>

main() {
   ... etc
`)
```

And for the filterscript specific code you can just use fmt.Sprintf and insert it conditionally with `%s`